app/src/components/menu.ts

```diff
@@ -286,6 +289,43 @@ export function createMenu({
     menuTemplate = [editMenu, viewMenu, windowMenu, helpMenu];
   }
 
+  try {
+    const bookmarkConfigPath = path.join(__dirname, '..', 'bookmarks.json');
+    if (fs.existsSync(bookmarkConfigPath)) {
+      const bookmarksConfig = JSON.parse(
+        fs.readFileSync(bookmarkConfigPath, 'utf-8'),
+      );
+      const bookmarksMenu: MenuItemConstructorOptions = {
+        label: bookmarksConfig['menuLabel'],
+        submenu: bookmarksConfig['bookmarks'].map((el) => {
+          if (el['type'] === 'link') {
+            let accelerator = null;
+            if ('shortcut' in el) {
+              accelerator = el['shortcut'];
+            }
+            return {
+              label: el['title'],
+              click: () => {
+                gotoUrl(el['url']);
+              },
+              accelerator: accelerator,
+            };
+          } else if (el['type'] === 'separator') {
+            return {
+              type: 'separator',
+            };
+          }
+        }),
+      };
+      menuTemplate.splice(menuTemplate.length - 2, 0, bookmarksMenu);
```

Comment:
Hmmmm, where will this insert the menu?

BTW, please attach to the PR a screenshot of the result.

Reply:
> Hmmmm, where will this insert the menu?

Looking at Safari and Firefox, the Bookmarks menu shows up between View and Window (as does History, but nativefier doesn't have that menu). So I aimed to put this new menu between View and Window as well.

<img width="574" alt="Screen Shot 2021-04-23 at 9 54 10 AM" src="https://user-images.githubusercontent.com/102904/115881939-43e33200-a41a-11eb-8edd-a31b9bd4dd21.png">
<img width="584" alt="Screen Shot 2021-04-23 at 9 54 17 AM" src="https://user-images.githubusercontent.com/102904/115881945-447bc880-a41a-11eb-8c55-5e152e7067a8.png">

> please attach to the PR a screenshot of the result.

Just added it to the PR description.